app/controllers/patients_controller.rb

```diff
@@ -54,6 +54,26 @@ def destroy
     end
   end
 
+  def sync_to_server
+    return redirect_back(fallback_location: root_path) unless @patient.unsynced?
+
+    host = ENV.fetch('SIMPLE_SERVER_HOST')
```

Comment:
not sure where or why

Reply:
not sure where and why

Reply:
You can create a service class which abstracts this logic. I would like to avoid adding this logic in a controller. 
We could also use the same service class in the sync task.

Reply:
yeah, shouldn't belong in the controller ideally.
will add it to the service class.

Reply:
updated